packages/common/editors/__tests__/CheckboxEditor.spec.js

```diff
@@ -1,27 +1,45 @@
 const React          = require('react');
 const ReactDOM = require('react-dom');
 const TestUtils      = require('react-dom/test-utils');
+const { mount } = require('enzyme')
+const sinon = require('sinon')
 const CheckboxEditor = require('../CheckboxEditor');
-const { mount } = require('enzyme');
 
 describe('CheckboxEditor', () => {
   let component;
   let componentWrapper;
+  let consoleErrorStub;
   let testColumn = {
     key: 'columnKey',
     onCellChange: function() {}
   };
 
   describe('Basic tests', () => {
+    afterEach(() => {
+      consoleErrorStub.restore()
+    })
+
     beforeEach(() => {
       spyOn(testColumn, 'onCellChange');
+      consoleErrorStub = sinon.stub(console, 'error')
       componentWrapper = mount(<CheckboxEditor
         value={true}
         rowIdx={1}
         column={testColumn}/>);
       component = componentWrapper.instance();
     });
 
+    it('should not send warning message to console.error', () => {
+      expect(consoleErrorStub.callCount).toBe(0)
```

Comment:
Because of the way _something_ is swallowing warning messages to console.error, this needs to be the first test. Otherwise the callCount would always be zero.